packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -31,6 +32,8 @@ import { ensure0x, spawnCmd, spawnCmdWithExitOnFailure } from '../lib/utils'
 
 const MonorepoRoot = resolvePath(joinPath(__dirname, '../..', '../..'))
```

Comment:
Mega-nit: The capitalization on this variable is bothering me
```suggestion
const monorepoRoot = resolvePath(joinPath(__dirname, '../..', '../..'))
```